### DISCUSSION TITLE
Make Chat box visible under scoreboard when spectating multiplayer

### DISCUSSION THREAD

**Discussion by TheFantasticLoki** [ppy/osu#16965] [Ideas] [answered]
Feature I want:
To be able to see the chat box under the scoreboard while spectating a multiplayer match like it is when you are actually playing in the match, you can access global chat with f8 when spectating but not the multiplayer chat channel.

Spectator mode with chat missing:
![osu_2022-02-22_18-03-07](https://user-images.githubusercontent.com/26889568/155245763-5aaadb0a-6dc3-4bd3-bca7-acccf44b4a47.png)

Ingame with chat visible under leaderboard:
![osu_2022-02-22_18-05-22](https://user-images.githubusercontent.com/26889568/155246060-93e3a06e-3b3c-49e4-bef7-12113d326043.png)

**Comment by TheFantasticLoki** [comment]
I feel like this may already be implemented to an extent just sort of bugged but not entirely sure that's why I made a discussion for it.

**Comment by smoogipoo** [comment]
Just to clarify, the primary issue here is that you can't see the multiplayer chat at all during spectator, is that right? Since it's no longer present in the global overlay.

Is having a standalone chat display at all times somewhere alongside the scoreboard something we want @peppy? Also not sure how this is going to translate into tournament, because we were originally discussing making multi-spectator use the tournament designs, which specifically don't show the chat in favour of the scores.

**Reply by peppy** [reply]
Yep, should be displayed just like it is during gameplay.

**Comment by smoogipoo** [comment, marked as answer]
Will be resolved by https://github.com/ppy/osu/pull/16985
